README.md

```diff
@@ -1,4 +1,4 @@
-# kali-packer :dragon:📦 #
+# kali-packer 💀📦🆒 #
```

Comment:
💀 --> 🐉 (just saying)
#CriticalPath

Reply:
Ah, something must have happened when I merged in cisagov/skeleton-packer-cool.  I didn't purposefully change the emojis.